.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: android
-jdk: oraclejdk8
+jdk: oraclejdk12
```

Comment:
what is the issue with jdk8?
anyways, please ensure all unit tests are passed. Seem that there are failures on Travis build.

Reply:
revert already